src/main/java/me/glasscrab/plentiful_harvest/Manager.java

```diff
@@ -39,29 +38,25 @@ public ItemStack makeSuperCrop(String name, Material material, List<String> lore
     }
 
     public boolean isFull(Player player) {
-        for (ItemStack checkItem : player.getInventory().getStorageContents()) {
-            if(checkItem == null) return false;
-            else if(checkItem.getItemMeta() == null) return false;
-        }
-
-        return true;
+        //Returns -1 if the inventory is full
+        return player.getInventory().firstEmpty() == -1;
     }
 
     public void giveSuperCrop(Player player, ItemStack superCrop) {
-        if (player.getInventory().firstEmpty() != -1) { //Returns -1 if the inventory is full
-            player.getInventory().addItem(superCrop);
+        if (isFull(player)) {
+            player.getWorld().dropItem(player.getLocation(), superCrop);//Drops to world
             return;
         }
-        player.getWorld().dropItem(player.getLocation(), superCrop);
+        player.getInventory().addItem(superCrop); // Gives item to player inventory
     }
 
-    public void giveDroppedItems(Player player, List<Item> droppedItems) {
+    /*public void giveDroppedItems(Player player, List<Item> droppedItems) {
         for(Item droppedItem : droppedItems) {
             droppedItem.remove();
             droppedItem.getItemStack().setAmount(droppedItem.getItemStack().getAmount()-1);
             player.getInventory().addItem(droppedItem.getItemStack());
         }
-    }
+    }*/
```

Comment:
Why is this commented? Can it just be removed?

Reply:
I didn't want to remove it just in case someone had an idea planned for it. I can remove it.